### PR TITLE
Check HTTPs is present in Local Network view

### PR DESCRIPTION
**Pull request by pablomendezroyo**

### CODE REVIEW

File: packages/admin-ui/src/pages/wifi/components/localProxying/LocalProxying.tsx [hunk 1]
```diff
@@ -60,6 +64,25 @@ export function LocalProxying(): JSX.Element {
...
   }
 
+  // Helper UI in case the HTTPs Portal is bad
+  if (dnpsRequest.data) {
+    const httpsPortalDnp = dnpsRequest.data.find(
+      dnp => dnp.dnpName === httpsPortalDnpName
+    );
+    if (
+      !httpsPortalDnp &&
+      dappnodeIdentity.internalIp !== dappnodeIdentity.ip
```
Comment (dapplion) [inline packages/admin-ui/src/pages/wifi/components/localProxying/LocalProxying.tsx:74]: This check needs way more explanation, you must add a big comment here. Is it even necessary tho to do this check? Consider:
- Receiving a boolean from the DAPPMANAGER backend for `isBehindRouter` or something
- Always should the banner

File: packages/admin-ui/src/pages/wifi/components/localProxying/LocalProxying.tsx [hunk 2]
```diff
@@ -124,15 +126,15 @@ export function LocalProxying(): JSX.Element {
           </div>
         </Card>
       ) : localProxyingStatus.isValidating ? (
-        <Loading steps={["Loading Local Network Proxy..."]} />
+        <Loading steps={["Loading Local Network Handler..."]} />
```
Comment (dapplion) [inline packages/admin-ui/src/pages/wifi/components/localProxying/LocalProxying.tsx:130]: What does it mean "handler" here?

File: packages/admin-ui/src/pages/wifi/components/localProxying/LocalProxying.tsx [hunk 3]
```diff
@@ -7,19 +7,43 @@ import { ReqStatus } from "types";
...
 import { StateBadge } from "pages/packages/components/StateBadge";
 import { MdWifi } from "react-icons/md";
 import { parseContainerState } from "pages/packages/components/StateBadge/utils";
+import Alert from "react-bootstrap/esm/Alert";
 import { LocalProxyingStatus } from "common";
+import { NavLink } from "react-router-dom";
 import LinkDocs from "components/LinkDocs";
 
 export function LocalProxying(): JSX.Element {
+  const dnpsRequest = useApi.packagesGet();
```
Comment (dapplion) [inline packages/admin-ui/src/pages/wifi/components/localProxying/LocalProxying.tsx:26]: Just inline this logic in the same component. Now you won't show any help text describing this feature. So the text says "You must install the HTTPs Portal to use this feature", but what feature is it?

File: packages/admin-ui/src/pages/wifi/components/localProxying/LocalProxying.tsx (hunk 2 above)
Comment (pablomendezroyo) [reply packages/admin-ui/src/pages/wifi/components/localProxying/LocalProxying.tsx:130]: back to proxy

File: packages/admin-ui/src/pages/wifi/components/localProxying/LocalProxying.tsx (hunk 3 above)
Comment (pablomendezroyo) [reply packages/admin-ui/src/pages/wifi/components/localProxying/LocalProxying.tsx:26]: done

**Review by dapplion — APPROVED** [review]

**Pull request merged**